interfaces/builtin/modem_manager.go

```diff
@@ -746,6 +746,9 @@ ACTION!="add|change|move", GOTO="mm_usb_device_blacklist_end"
 SUBSYSTEM!="usb", GOTO="mm_usb_device_blacklist_end"
 ENV{DEVTYPE}!="usb_device",  GOTO="mm_usb_device_blacklist_end"
 
+# Telegesis zigbee dongle
+ATTRS{idVendor}=="10c4", ATTRS{idProduct}=="0003", ENV{ID_MM_DEVICE_IGNORE}="1"
```

Comment:
According to http://www.linux-usb.org/usb.ids this is 

vendor: 10c4  Cygnal Integrated Products, Inc.
product: 0003  CommandIR

This doesn't look like zigbee to me, can you confirm please?

Reply:
Cygnal was acquired by Silicon Labs, which sells these telegesis modules:
https://www.silabs.com/products/wireless/mesh-networking/telegesis-modules-gateways

Reply:
I pulled the Windows driver for this module from here: https://www.silabs.com/documents/public/software/telegesis-usb-driver.zip and there's this section in the `*.inf` file:
```

[SiLabs]
%USB\VID_10C4&PID_8293.DeviceDesc%=silabser.Dev, USB\VID_10C4&PID_8293
```
This would suggest that the Vendor ID is correct, but product ID is different from what you have. This would also explain why there is a different module listed in the linux USB page. The udev match line should probably look like this then:
```
ATTRS{idVendor}=="10c4", ATTRS{idProduct}=="8293", ENV{ID_MM_DEVICE_IGNORE}="1"
```
Can you check if it works for you?

Reply:
I can confirm that the communication from the customer is that the VID/PID we are interested in is 10c4:0003